File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Javascript/Excercises/program-5.js
// 5-Escribir un programa que pida al usuario una cadena de caracteres y muestre por pantalla si contiene la letra "r".


let userInput = prompt('Ingrese una cadena de caracteres: ')

if (userInput.includes("r")) {
    console.log(`${true}, La cadena ${userInput} si contiene la letra "r"`)
} else {
    console.log(`${false}, La cadena ${userInput} no contiene la letra "r"`)
}

File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Python/Excercises/program-5.py
# 5-Escribir un programa que pida al usuario una cadena de caracteres y muestre por pantalla si contiene la letra "r".


userInput = input("Ingresa una cadena de caracters: ")

if "r" in userInput:
    print(f'{True}, La cadena ({userInput}) si contiene la laetra "r"');
else:
    print(f'{False}, La cadena ({userInput}) no contiene la laetra "r"');